unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/PrivateChatWindow/PrivateChatWindowController.cs

```diff
@@ -311,4 +325,36 @@ private void HandleChatInputTriggered(DCLAction_Trigger action)
         if (!View.IsActive) return;
         chatHudController.FocusInputField();
     }
+
+    internal void RequestOldConversations()
+    {
+        if (isRequestingOldMessages)
+            return;
+
+        isRequestingOldMessages = true;
+        View?.SetOldMessagesLoadingActive(true);
+
+        if (!lastTimestampRequestedByUser.ContainsKey(ConversationUserId))
+            lastTimestampRequestedByUser.Add(ConversationUserId, DateTimeOffset.UtcNow.ToUnixTimeMilliseconds());
+
+        List<ChatMessage> currentPrivateMessages = chatController.GetAllocatedEntries()
+            .Where(x => (x.sender == ConversationUserId || x.recipient == ConversationUserId) && x.messageType == ChatMessage.Type.PRIVATE)
+            .ToList();
+
+        if (currentPrivateMessages.Count > 0)
+        {
+            long minTimestamp = (long)currentPrivateMessages.Min(x => x.timestamp);
+            ChatMessage aaa = currentPrivateMessages.FirstOrDefault(x => x.timestamp == (ulong)minTimestamp);
```

Comment:
delete

Reply:
Ups, it was a debugging code that I forgot to remove. Done in https://github.com/decentraland/unity-renderer/pull/2484/commits/cac18537f01ae13f8878e67ab7276d79fdc08640